.github/workflows/test.yml

```diff
@@ -45,6 +45,8 @@ jobs:
         run: sbt ++${{ matrix.scala }} test
       - name: Binary compatibility
         run: sbt ++${{ matrix.scala }} mimaReportBinaryIssues
+      - name: Mill Compile
+        run: mill _[${{ matrix.scala }}].compile
```

Comment:
This increases the CI latency. Could you make this a parallel build step please? That way people will get quicker feedback on their PRs.

As an example you could have a look at how I did that for treadle:
https://github.com/chipsalliance/treadle/blob/master/.github/workflows/test.yml#L44